### PR TITLE
add exclude-tags-rexec option

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

File: changelog.sh [hunk 1]
```diff
@@ -85,6 +85,11 @@ if [ ! -z "$INPUT_GITHUB_API" ]; then
   ARG_GITHUB_API="--github-api $INPUT_GITHUB_API"
 fi
 
+ARG_EXCLUDE_TAGS_REGEX=""
+if [ ! -z "$ARG_EXCLUDE_TAGS_REGEX" ]; then
+  ="--exclude-tags-regex $ARG_EXCLUDE_TAGS_REGEX"
```
Comment (charmixer) [inline changelog.sh]: Something went wrong here.

ARG left hand side INPUT right hand side.

Comment (rndmh3ro) [reply changelog.sh]: Thansk, fixed.

File: changelog.sh [hunk 2]
```diff
@@ -85,6 +85,11 @@ if [ ! -z "$INPUT_GITHUB_API" ]; then
   ARG_GITHUB_API="--github-api $INPUT_GITHUB_API"
 fi
 
+ARG_EXCLUDE_TAGS_REGEX=""
+if [ ! -z "$ARG_EXCLUDE_TAGS_REGEX" ]; then
```
Comment (charmixer) [inline changelog.sh]: if and assigned value should be `$INPUT_EXCLUDE_TAGS_REGEX`

Comment (rndmh3ro) [reply changelog.sh]: Fixed!

**Pull request merged**